pdfpreview.module

```diff
@@ -34,13 +34,12 @@ function pdfpreview_file_delete(EntityInterface $file) {
 }
 
 /**
- * Implements hook_imagemagic_arguments_alter().
+ * Implements hook_imagemagick_arguments_alter().
  */
 function pdfpreview_imagemagick_arguments_alter(ImagemagickExecArguments $arguments, $command) {
   $i = $arguments->findArgument('-pdfpreview');
   if ($i !== FALSE) {
     $arguments->removeArgument($i);
-    $arguments->setSourceLocalPath($toolkit->getSourceLocalPath() . '[0]');
```

Comment:
Hi @mstrelan,

Thank you for porting this module to Drupal 8.

The highlighted line above causes PDFPreview to not work properly for me. In case this was added in order to select the first page/frame, this seems to be set by default in `\Drupal\imagemagick\Plugin\ImageToolkit\ImagemagickToolkit::convert()`.

I am using `drupal/imagemagick:2.2.0` version.